spec/unit/provider/service/redhat_spec.rb

```diff
@@ -4,6 +4,10 @@
          if: Puppet.features.posix? && !Puppet::Util::Platform.jruby?do
   let(:provider_class) { Puppet::Type.type(:service).provider(:redhat) }
 
+  before(:all) do
```

Comment:
Thank you @IrimieBogdan ! Could you add a comment to the code here explaining why we have to do this? I can see someone getting really confused a year from now as to why this is needed :)

Reply:
Added a comments describing the fix and why it is required.